pkg/search/postgres/sac_test.go

```diff
@@ -4,23 +4,58 @@ import (
 	"context"
 	"testing"
 
+	v1 "github.com/stackrox/rox/generated/api/v1"
 	"github.com/stackrox/rox/generated/storage"
 	"github.com/stackrox/rox/pkg/auth/permissions"
 	"github.com/stackrox/rox/pkg/sac"
 	"github.com/stretchr/testify/assert"
+	"google.golang.org/protobuf/proto"
 )
 
 func TestGetReadWriteSACQuery(t *testing.T) {
 	ctx := sac.WithGlobalAccessScopeChecker(context.Background(), createTestReadMultipleResourcesSomeWithNamespaceScope(t))
 	got, err := GetReadWriteSACQuery(ctx, metadata("Cluster", permissions.ClusterScope))
-	assert.Equal(t, `base_query:<match_field_query:<field:"Cluster ID" value:"\"clusterID\"" > > `, got.String())
+	expectedQuery := &v1.Query{
+		Query: &v1.Query_BaseQuery{
+			BaseQuery: &v1.BaseQuery{
+				Query: &v1.BaseQuery_MatchFieldQuery{
+					MatchFieldQuery: &v1.MatchFieldQuery{
+						Field: "Cluster ID",
+						Value: "\"clusterID\"",
+					},
+				},
+			},
+		},
+	}
+	assert.True(t, proto.Equal(expectedQuery, got))
 	assert.NoError(t, err)
+
 	got, err = GetReadWriteSACQuery(ctx, metadata("Namespace", permissions.NamespaceScope))
-	assert.Equal(t, `base_query:<match_none_query:<> > `, got.String())
+	expectedQuery = &v1.Query{
+		Query: &v1.Query_BaseQuery{
+			BaseQuery: &v1.BaseQuery{
+				Query: &v1.BaseQuery_MatchNoneQuery{
+					MatchNoneQuery: &v1.MatchNoneQuery{},
+				},
+			},
+		},
+	}
```

Comment:
nit (code duplication): The expected query for match none query could be extracted to a file variable.